### PR TITLE
Chapter 4.7.1: fix asciidoc table syntax.

### PR DESCRIPTION
**Pull request by baranovskiykonstantin**

### CODE REVIEW

File: src/eeschema/eeschema_general_top_toolbar.adoc [hunk 1]
```diff
@@ -256,30 +256,30 @@ in a few components.
 
 This methods are illustrated below.
```
Comment (heitorPB) [inline src/eeschema/eeschema_general_top_toolbar.adoc]: Shouldn't this sentence start with `These` instead of `This`?

Comment (baranovskiykonstantin) [reply src/eeschema/eeschema_general_top_toolbar.adoc]: Yes, it should. My fault.

**Comment by tijuca** [comment]
Please merge this change after tagging the release 5.1.4 as it breaks potentially existing translations.

**Comment by nickoe** [comment]
Thank you all.

**Pull request merged**